src/DigitalSubscriberAppBanner.test.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { DigitalSubscriberAppBanner } from './DigitalSubscriberAppBanner';
+
+describe('DigitalSubscriberAppBanner', () => {
+    describe('when a button is clicked', () => {
```

Comment:
I guess we'll test the impression event separately?

Reply:
This is an interesting comment! As of this PR the _impression_ is still handled entirely by the platform. I've been mulling over the best way of handling this. I think if we want to move to the component, as we're doing with clicks, then a separate PR would be good.

One thing that's got me not 100% convinced about this is that in the case of a banner rendering and VIEWing are 1:1, but for other slots that might not be the case. For example an epic VIEW only happens when it enters the viewport potentially after it's been rendered. It may never be viewed at all. I don't know if the component would have enough information to make that decision, I'd like to try it out. But I'd rather not set a pattern without testing it if that pattern isn't going to work for us in future scenarios.

Reply:
I like the reasoning here 👍